### PR TITLE
Add iosxr-7.0.2 to nodesets

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gosriniv@redhat.com>

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: zuul.d/project.yaml
Comment (pabelanger) [inline zuul.d/project.yaml:7]: there is no need to delete these jobs, they will only run is files specific to them change.

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml
Comment (pabelanger) [inline playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml:39]: we should revert this back to admin, so we don't have to have 2 playbooks for iosxr. This likely means, re-rolling the iosxr images to default to admin.

File: playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml
Comment (pabelanger) [inline playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml:59]: lets try not to make this dynamic. if the password is the only issue, lets respin

File: playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml
Comment (pabelanger) [inline playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml:25]: so, we shouldn't need this. As our playbook adds that info on the controller side.

Can you setup an autohold on failure so I can look.  I suspect something in the controller is regenerating the host keys for some reason

File: playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml
Comment (GomathiselviS) [reply playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml:39]: @pabelanger iosxr7.0.2 has restrictions on minimum password length which is 6. So we will not be able to use admin. I shall add a check here, for 7.0.2

Comment (pabelanger) [reply playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml]: k, if that is the case we can move to inventory vars for, which are loaded depending on which image.

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Review by ashwini-mhatre — APPROVED** [review]

File: playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml
Comment (pabelanger) [inline playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml:37]: okay, lets add this back for now. Adding a NOTE here to explain what we think is happening.

File: playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml
Comment (pabelanger) [reply playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml:66]: lets drop when statement

File: playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml
Comment (pabelanger) [inline playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml:80]: can we drop when?

File: playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml
Comment (pabelanger) [inline playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml:93]: same

File: zuul.d/nodesets.yaml
Comment (pabelanger) [inline zuul.d/nodesets.yaml:597]: this likely all can remove, we mostly want to test against devel right now.  Given we didn't target our old collections for this version of IOSXR

**Comment by trishnaguha** [comment]
@ashwini-mhatre @Gomathiselvi-S What are we waiting on for the upgrade?

**Comment by ashwini-mhatre** [comment]
> @ashwini-mhatre @Gomathiselvi-S What are we waiting on for the upgrade?

once https://github.com/ansible-collections/cisco.iosxr/pull/130 this PR get approval we can merge this. I already got approval from rohit. waiting for approval from ganesh

**Pull request merged**
